src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/ActionPlanController.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.controller
+
+import org.springframework.http.ResponseEntity
+import org.springframework.security.oauth2.server.resource.authentication.JwtAuthenticationToken
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto.CreateActionPlanDTO
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto.SentReferralDTO
+
+@RestController
+class ActionPlanController() {
+
+  @PostMapping("/draft-actionPlan/create")
```

Comment:
capitalization here is funky. i think `draft-action-plan` is more consistent.

the general pattern for object creation in this api is just `POST /object`, so we can remove the create portion of the path.

Reply:
Will do